redwood-layout-compose/src/commonMain/kotlin/app/cash/redwood/layout/compose/LayoutScopes.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.layout.compose
+
+import androidx.compose.runtime.Stable
+import app.cash.redwood.Modifier
+
+/**
+ * Equivalent to `modifier.grow(value).shrink(value)`.
+ *
+ * Call this in [ColumnScope] or [RowScope].
```

Comment:
Unfortunately the compiler doesn’t enforce this.

Reply:
We'll be able to enforce this once context receivers are stable

Reply:
Let's file an issue to remember. Or leave a TODO to automatically create one.

Reply:
https://github.com/cashapp/redwood/issues/1580